code/modules/reagents/chemistry_reagents/stims.dm

```diff
@@ -0,0 +1,164 @@
+
+/datum/reagent/stimulant
+	reagent_state = LIQUID
+	custom_metabolism = AMOUNT_PER_TIME(1, 1 MINUTES) // Consumes 1 unit per minute.
+	overdose = LOWH_REAGENTS_OVERDOSE
+	overdose_critical = LOWH_REAGENTS_OVERDOSE_CRITICAL
+	chemclass = CHEM_CLASS_NONE
+	flags = REAGENT_SCANNABLE | REAGENT_TYPE_STIMULANT
+	var/jitter_speed = 0.3 SECONDS
+	var/jitter_per_amount = 2
+	var/jitter = 2
+
+/datum/reagent/stimulant/on_mob_life(mob/living/M, alien, delta_time)
+	. = ..()
+	// Stimulants drain faster for each stimulant in the drug.
+	// Having 2 stimulants means the duration will be 2x shorter, having 3 will be 3x shorter, etc
+	if(holder)
+		for(var/i in holder.reagent_list)
+			if(i == src)
+				continue
+
+			var/datum/reagent/R = i
+			if(R.flags & REAGENT_TYPE_STIMULANT)
+				holder.remove_reagent(R, custom_metabolism)
+
+	// We multiply delta_time by 1.5 so that it looks like it is consistent.
```

Comment:
is this comment still relevant? i don't see usage of delta_time

Reply:
I have no idea, I just moved it here from another file.